Components/Molecules/SignUpContainer.js

```diff
@@ -34,6 +35,13 @@ const SignUpContainer = ({
             <View>{children}</View>
             <Button label={nextLabel || "Next"} onPress={next} full primary />
             <Button label="Back" onPress={back} full />
+            <Button
```

Comment:
The SignUpContainer is used by the SignUpEmail, SignUpName, and SignUpPassword pages, which means Forgot Password is showing up  there. 
![Screenshot 2020-05-17 16 17 15](https://user-images.githubusercontent.com/34425825/82159126-0e24e900-985a-11ea-8c88-6323b0050c22.png)

Reply:
good catch, added a condition check to render this button only on Login screen